source/com.google.android.gms/play-services-basement/Additions/Additions.cs

```diff
@@ -1,5 +1,10 @@
-﻿[assembly: Android.App.UsesLibrary("org.apache.http.legacy", Required=false)]
+﻿using Android.OS;
+
+[assembly: Android.App.UsesLibrary("org.apache.http.legacy", Required=false)]
 
 namespace Android.Gms.Common
-{
+{
+	namespace Android.Gms.Common.Apis
+	{
+    }
```

Comment:
This might introduce a bad namespace: `Android.Gms.Common.Android.Gms.Common.Apis`

Reply:
True. Sorry. I messed up something. Fixed